### PR TITLE
Add blog post: Status Update 2018-09-15

### PR DESCRIPTION
**Pull request by ubruhin**

### CODE REVIEW

File: content/blog/2018-09-15_status_update.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+unresolved for more than three years (it was our 6th issue!), we decided to
+start a [design contest at 99designs](http://99d.me/c/iab2) to get a new logo.
+And here's the winner design:
+
+![New Logo](/img/new_logo.png)
+
+Sure, it's quite similar to the old logo, just nicer ;-) Probably that's because
+I already liked the old logo, and maybe also a little bit because it was (too)
+hard for me to go with a completely different logo than the one we had since
+several years... Anyway, I hope you like the new logo as I do :)
```
Comment (dbrgn) [inline content/blog/2018-09-15_status_update.md]: "as much as I do"

Comment (ubruhin) [reply content/blog/2018-09-15_status_update.md]: Fixed. Thanks for reviewing :+1:

**Comment by dbrgn** [comment]
Don't forget to post it to Patreon :)

File: content/blog/2018-09-15_status_update.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+Our logo was originally downloaded from a free icon portal. Although it was
+legal (thanks to its permissive license), it's not a good idea to use a free
+icon as a logo. Because we didn't have the copyright on it, it was not
+guaranteed to be exclusively used by us (other projects could use the same icon
+too).
+
+As the [corresponding issue](https://github.com/LibrePCB/LibrePCB/issues/6) was
+unresolved for more than three years (it was our 6th issue!), we decided to
+start a [design contest at 99designs](http://99d.me/c/iab2) to get a new logo.
+And here's the winner design:
```
Comment (rnestler) [inline content/blog/2018-09-15_status_update.md]: "winner design" sounds strange to me. Maybe "winning design"?

File: content/blog/2018-09-15_status_update.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+![New Logo](/img/new_logo.png)
+
+Sure, it's quite similar to the old logo, just nicer ;-) Probably that's because
+I already liked the old logo, and maybe also a little bit because it was (too)
+hard for me to go with a completely different logo than the one we had since
```
Comment (rnestler) [inline content/blog/2018-09-15_status_update.md]: the one we had *for* several years

File: content/blog/2018-09-15_status_update.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+made the board editor really usable (it fixed many annoying bugs).
+
+
+### Next Steps
+
+With all the changes from above, our
+[milestone 0.1](https://github.com/LibrePCB/LibrePCB/milestone/1) is nearly
+complete. That means that we are now feature-complete for our first release!
+
+So you can expect the first official stable release very soon :-)
```
Comment (rnestler) [inline content/blog/2018-09-15_status_update.md]: The 0.1 release isn't really stable is it?

File: content/blog/2018-09-15_status_update.md (hunk 2 above)
Comment (ubruhin) [reply content/blog/2018-09-15_status_update.md]: Fixed in 131d084d1a0f9af532b076da9a9ae1a2844e87da

File: content/blog/2018-09-15_status_update.md (hunk 3 above)
Comment (ubruhin) [reply content/blog/2018-09-15_status_update.md]: Fixed in 131d084d1a0f9af532b076da9a9ae1a2844e87da

**Pull request merged**